crates/hud/src/hud.rs

```diff
@@ -0,0 +1,124 @@
+use bevy::asset::{AssetServer, Handle};
+use bevy::prelude::{BuildChildren, ChildBuilder, Color, Commands, default, NodeBundle, ResMut};
+use bevy::text::Font;
+use bevy::ui::{AlignItems, AlignSelf, BackgroundColor, FlexDirection, JustifyContent, PositionType, Size, Style, UiRect, Val};
+
+use de_core::objects::BuildingType;
+
+use crate::interaction::BuildBuildingButton::BuildBuilding;
+use crate::universal_ui::{create_button_bundle, create_detail_node_bundle, create_side_node_bundle, create_text_in_button_bundle, universal_style_full};
+
+pub fn spawn_hud(
+    mut commands: Commands,
+    asset_server: ResMut<AssetServer>,
+) {
+    let font: Handle<Font> = asset_server.load("fonts/Fira_Mono/FiraMono-Medium.ttf");
```

Comment:
I hoped to have all the low-level / building-block UI functionality inside the UI crate and then use it here to build up the HUD.